rpcs3/Emu/RSX/VK/VKDMA.cpp

```diff
@@ -277,6 +277,10 @@ namespace vk
 		{
 			allow_host_buffers = true;
 		}
+#elif defined(__FreeBSD__)
+		// Anything running on AMDGPU kernel driver will not work due to the check for fd-backed memory allocations
+		// Requires root privilege on FreeBSD on Intel
```

Comment:
root is specific to `mesa-devel` as `mesa-dri` package lacks https://github.com/freebsd/freebsd-ports/commit/058e5ba0ff29ae64383b8392f23facad616be8c9. With vanilla Mesa VK_EXT_external_memory_host simply fails unless user hides the extension from RPCS3 e.g., via a Vulkan layer.